src/main/controllers/rpc-handler.js

```diff
@@ -1012,9 +1036,18 @@ module.exports = function(app) {
 	};
 
 	controller.prototype.getWalletTokens = function(event, actionId, actionName, args) {
-		WalletToken.findByWalletId(args.walletId)
-			.then(data => {
-				app.win.webContents.send(RPC_METHOD, actionId, actionName, null, data);
+		Wallet.findById(args.walletId)
+			.then(wallet => {
+				WalletToken.findByWalletId(wallet.id)
+					.then(data => {
```

Comment:
wallet tokens can be loaded with eager:
`Wallet.findById(id).eager('tokens')`
Than can be used as:
`wallet.tokens`

Reply:
@sk91 I tried that but it is not returning the tokens and instead undefined. I think we can go back to it when we refactor the token persistence. Thoughts? Cheers

Reply:
@rodrigopavezi strange, ok will check it out in a separate pull request